client/components/chart/index.js

```diff
@@ -33,20 +33,23 @@ d3FormatDefaultLocale( {
 	currency: [ decodeEntities( get( wcSettings, 'currency.symbol', '' ) ), '' ],
 } );
 
-function getOrderedKeys( props ) {
+function getOrderedKeys( props, previousOrderedKeys = [] ) {
 	const updatedKeys = [
 		...new Set(
 			props.data.reduce( ( accum, curr ) => {
 				Object.keys( curr ).forEach( key => key !== 'date' && accum.push( key ) );
 				return accum;
 			}, [] )
 		),
-	].map( key => ( {
-		key,
-		total: props.data.reduce( ( a, c ) => a + c[ key ].value, 0 ),
-		visible: true,
-		focus: true,
-	} ) );
+	].map( key => {
+		const previousKey = find( previousOrderedKeys, item => key === item.key );
```

Comment:
When there is a ES6 method equivalent to a lodash method, I think using ES6 makes it easy for contributors who might not be used to lodash to understand the code. What do you think?
```suggestion
		const previousKey = previousOrderedKeys.find( item => key === item.key );
```